kinto/core/utils.py

```diff
@@ -443,3 +443,240 @@ def instance_uri(request, resource_name, **params):
     """Return the URI for the given resource."""
     return strip_uri_prefix(request.route_path('%s-record' % resource_name,
                                                **params))
+
+
+class JsonPatch(object):
+    """
+    Process a JSON Patch over a record.
+    """
```

Comment:
This is too big to be included like that.

I suggest that we rely on an external library instead (like https://github.com/stefankoegl/python-json-patch) and here keep a small helper only.